wrap/.gitignore

```diff
@@ -1,2 +1,5 @@
 __pycache__/
 .vscode/
+*build*
+*.egg-info
+cmake/PybindWrap.cmake
```

Comment:
Hmm, why is this file here?

Reply:
This PR introduces a `cmake/PybindWrap.cmake.in` file so ignoring the generated file. :slightly_smiling_face: